redash/query_runner/dgraph.py

```diff
@@ -0,0 +1,160 @@
+import json
+
+try:
+    import pydgraph
+    enabled = True
+except ImportError:
+    enabled = False
+
+from redash.query_runner import BaseQueryRunner, register
+from redash.utils import json_dumps, json_loads
+
+
+def to_string(s):
+    """From https://github.com/vinay20045/json-to-csv"""
+    try:
+        return str(s)
+    except:
+        # Change the encoding type if needed
+        return s.encode('utf-8')
+
+
+def reduce_item(reduced_item, key, value):
+    """From https://github.com/vinay20045/json-to-csv"""
+    # Reduction Condition 1
+    if type(value) is list:
+        i = 0
+        for sub_item in value:
```

Comment:
You can use `enumerate` here to avoid the need to manually keep track of `i`.

```suggestion
        for i, sub_item in enumerate(value):
```

Reply:
done